terraform-tests/pubsub_test.go

```diff
@@ -20,13 +20,22 @@ var _ = Describe("PubSub", Label("pubsub-terraform"), Ordered, func() {
 
 	BeforeAll(func() {
 		defaultVars = map[string]any{
-			"credentials":       googleCredentials,
-			"project":           googleProject,
-			"labels":            map[string]string{"label1": "value1"},
-			"topic_name":        "test-topic-name",
-			"subscription_name": "",
-			"ack_deadline":      10,
-			"push_endpoint":     "",
+			"credentials":                      googleCredentials,
+			"project":                          googleProject,
+			"labels":                           map[string]string{"label1": "value1"},
+			"topic_name":                       "test-topic-name",
+			"subscription_name":                "",
+			"ack_deadline":                     10,
+			"push_endpoint":                    "",
+			"topic_message_retention_duration": "",
+			"topic_kms_key_name":               "",
+			"subscription_message_retention_duration":   "",
```

Comment:
The formatting here seems to be a bit off

Reply:
I know, but when formatting, the fmt tool does not change anything :thinking: